src/pages/installing/installing.md

```diff
@@ -64,14 +58,14 @@ To add the pull secret to the Openshift global pull secret:
 
 1. Create a base64 encoded string with the registry userid and password as it aligns with your access method.
```

Comment:
Remove everything in this section from here on. This is specific to the IBM registry, not Azure.